### PR TITLE
✨️ Add hyperlinks, reviewers, assignees etc.

### PR DESCRIPTION
**Pull request by smolijar**
Example output:
```
Project                                    Id Assignees Reviewers Title                                                  
------------------------------------------ -- --------- --------- ------------------------------------------------------ 
AckeeCZ/tslint-config-ackee                7  vlasy     vlasy     Add no-return-await                                    
grissius/sheep-a-changelog                 14                     Feature/generate                                       
```

### CODE REVIEW

**Comment by loukotal** [comment]
Nice, thanks! 

:shipit:

File: Cargo.toml [hunk 1]
```diff
@@ -9,9 +9,9 @@ description = "MR. Dog groups and watches merge requests across all your *hubs"
 [dependencies]
 async-trait = "0.1.64"
 clap = { version = "4.1.1", features = ["derive"] }
+colored = "2.0.0"
```
Comment (loukotal) [inline Cargo.toml]: is this used? 🤔 or is "TBU"? 🐱

Comment (smolijar) [reply Cargo.toml]: I meant to use it but forgoth, will remove until we have a use for it, es ques moa

Comment (smolijar) [reply Cargo.toml]: Removed

**Comment by smolijar** [comment]
Removed colored and fixed pipeline, merging.

**Pull request merged**